components/layout_2020/flow/inline/text_run.rs

```diff
@@ -64,16 +67,18 @@ enum SegmentStartSoftWrapPolicy {
     FollowLinebreaker,
 }
 
-#[derive(Debug)]
+#[derive(Debug, MallocSizeOf)]
 pub(crate) struct TextRunSegment {
     /// The index of this font in the parent [`super::InlineFormattingContext`]'s collection of font
     /// information.
     pub font_index: usize,
 
     /// The [`Script`] of this segment.
+    #[ignore_malloc_size_of = "unicode-script type"]
```

Comment:
Please add `malloc_size_of_is_0` to `components/malloc_size_of/lib.rs` for these types.